plugins/remember.py

```diff
@@ -104,14 +105,68 @@ def forget(inp, chan='', db=None):
         return "I don't know about that."
 
 
-@hook.regex(r'^\? ?(.+)')
+def get_page(data, start_idx, min_page_len, max_page_len):
+    page_data = data[start_idx:start_idx + max_page_len]
+    last_comma_idx = page_data.rfind(',')
+
+    if last_comma_idx == -1 or last_comma_idx < min_page_len:
+        ret = data[start_idx:max_page_len + start_idx]
+        return ret, len(ret) + start_idx
+
+    end_idx = last_comma_idx + start_idx
+    return data[start_idx:end_idx], end_idx
+
+
+def get_pages(data, min_page_len, max_page_len):
+    result = {}
```

Comment:
my python is rusty, but shouldn't this be `[]`, since you want to return an array?